Assignment3/DLDLsensation-x150279/Assignment3.txt

```diff
@@ -0,0 +1,569 @@
+1)Get all the properties that can be applied to instances of the Politician class (<http://dbpedia.org/ontology/Politician>)
+
+PREFIX pol: <http://dbpedia.org/ontology/Politician> 
+select distinct ?p
+where {
+?i a pol: .
```

Comment:
I recommend you to divide more clearly between PREFIX (http://dbpedia.org/ontology/) and the final part of the URI (Politician) instead of using a prefix pol and use pol: